<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_OgeWiFixPosition</class>
 <widget class="QWidget" name="C_OgeWiFixPosition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>161</width>
    <height>55</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C_OgeWiFixPosition</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeGbxHover" name="pc_GroupBox">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="stw_opensyde_gui_elements::C_OgeLabDoubleClick" name="pc_LabelIcon">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="stw_opensyde_gui_elements::C_OgeLabDoubleClick" name="pc_LabelTitle">
        <property name="font">
         <font>
          <family>Segoe UI</family>
         </font>
        </property>
        <property name="text">
         <string>TITLE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="stw_opensyde_gui_elements::C_OgePubIconOnly" name="pc_PushButtonMax">
        <property name="minimumSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgePubIconOnly</class>
   <extends>QPushButton</extends>
   <header>C_OgePubIconOnly.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeGbxHover</class>
   <extends>QGroupBox</extends>
   <header>C_OgeGbxHover.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabDoubleClick</class>
   <extends>QLabel</extends>
   <header>C_OgeLabDoubleClick.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
